src/main/java/cd/connect/openapi/DartV3ApiGenerator.java

```diff
@@ -33,10 +32,11 @@
 import java.util.Set;
 import java.util.stream.Collectors;
 
-public class DartV3ApiGenerator extends DartClientCodegen implements CodegenConfig {
+public class DartV3ApiGenerator extends DartClientCodegen {
   private static final Logger log = LoggerFactory.getLogger(DartV3ApiGenerator.class);
   private static final String LIBRARY_NAME = "dart2-api";
   private static final String DART2_TEMPLATE_FOLDER = "dart2-v3template";
+  @SuppressWarnings("unused") // mustache uses this (?)
```

Comment:
this one I'm a bit confused about, the compiler remarks that the variable is unused, but searching the whole project I find this string in the mustache templates, so I assumed they reference this somehow by a mechanism I don't understand? 😅

Reply:
It would be safe to remove them, because the prefix is used "x-dart-produces-" and the "raw" is used when determining its an octet stream. It adds json, xml, etc as well.

Reply:
My recommendation is to delete it :-)

Reply:
ah, now I see it. Ok, will delete it